### PR TITLE
Use FindUUID.cmake from cmake-modules to find the UUID libraries

### PR DESCRIPTION
**Pull request by esteve**
Solves #26

Depends on ros/cmake_modules@b45081a0cb562d05214aab1965fbf7b09f613812

### CODE REVIEW

**Comment by esteve** [comment]
@rcodddow Could you test this branch? You'll need a recent copy of ros/cmake_modules in your workspace (at least ros/cmake_modules@b45081a0cb562d05214aab1965fbf7b09f613812)

File: nodelet/package.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
   <build_depend>bondcpp</build_depend>
   <build_depend>boost</build_depend>
+  <build_depend>cmake_modules</build_depend>
```
Comment (dirk-thomas) [inline nodelet/package.xml]: Needs a version dependency.

**Comment by rcodddow** [comment]
nope still the same issue

**Comment by esteve** [comment]
@rcodddow did the previous fix in nodelet_core work for you? Could you paste the console output when trying to build bond_core?

**Comment by rcodddow** [comment]
looks like I spoke too soon. I started with a new ros workspace and didn't have https://github.com/ros/bond_core/pull/8.

**Comment by esteve** [comment]
@rcodddow I just tested this on Mavericks with the cmake_modules fix, I believe this issue might have arisen due to a recent change in Homebrew. In any case, the previous code wasn't cross-platform.

**Comment by dirk-thomas** [comment]
You should consider squashing your commits in the future. Otherwise it becomes very painful when you need to backport changes.

**Pull request merged**